Cannot build i3status-rust without pulse since commit: 4df33cc2483d63e62d2ac4bd790a26dfe128b781

I can not build i3status-rust without pulse since commit: 4df33cc2483d63e62d2ac4bd790a26dfe128b781

```
$ cargo build --release --no-default-features
...
   Compiling i3status-rs v0.14.1 (/usr/local/src/i3status-rust)
error[E0425]: cannot find function `min` in this scope
   --> src/blocks/sound.rs:129:13
    |
129 |             min(new_vol, vol_cap)
    |             ^^^ not found in this scope
    |
help: consider importing one of these items
    |
24  | use core::cmp::min;
    |
24  | use std::cmp::min;
    |

error: aborting due to previous error

For more information about this error, try `rustc --explain E0425`.
error: could not compile `i3status-rs`.
```

_Originally posted by @NOTMY in https://github.com/greshake/i3status-rust/issues/752#issuecomment-706395952_

--- a/src/blocks/sound.rs
+++ b/src/blocks/sound.rs
@@ -13,7 +13,6 @@ use {
     crossbeam_channel::unbounded,
     lazy_static::lazy_static,
     std::cell::RefCell,
-    std::cmp::min,
     std::collections::HashMap,
     std::convert::{TryFrom, TryInto},
     std::ops::Deref,
@@ -21,7 +20,7 @@ use {
     std::sync::Mutex,
 };
 
-use std::cmp::max;
+use std::cmp::{max, min};
 use std::collections::BTreeMap;
 use std::io::Read;
 use std::process::{Command, Stdio};

